### DISCUSSION TITLE
Lost email

### DISCUSSION THREAD

**Discussion by ghost** [community/community#32660] [Other Feature Feedback, Questions, & Ideas] [answered]
Recently I created a second github account with an email address for which I lost the password, and it was the only one that was paired with the account. I had conected with this account on github desktop. Result I can no longer connect to the web but I am still connected to github desktop. Please help!

**Comment by airtower-luna** [comment, marked as answer]
Account recovery is something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. Keep in mind though that [multiple free accounts for one person are against the GitHub ToS](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#3-account-requirements) (except one machine account), I'm not sure if that could get your other account into trouble. Good luck!

**Reply by ghost** [reply]
thanks ^^
